themes/default/content/docs/pulumi-cloud/deployments/drift.md

```diff
@@ -12,13 +12,13 @@ menu:
 
 Drift Detection is the process of identifying changes in the actual state of your cloud environment that deviate from the expected state stored in Pulumi Cloud. This deviation can occur for various reasons, including manual adjustments made directly in the cloud provider’s console, unintended consequences of scripts, or unauthorized changes.
 
-In order to detect and/or remediate drift via Pulumi Deployments, you are first required to configure deployment settings for your stack.
+To use Drift Detection and Remediation via Pulumi Deployments, you are first required to configure deployment settings for your stack.
 
-You can also run a remediate drift operation, which will run a `pulumi up --refresh` to treat the pulumi program as the source of truth and override what is in the cloud.
+You can also run a remediate drift operation, which will run a `pulumi up --refresh` to treat the Pulumi program as the source of truth and override what is in the cloud.
 
-## Runing Drift detection from the CLI
+## Running Drift Detection from the CLI
 
-Any Preview of a Refresh is considered a drift detection run in Pulumi Cloud. Running drift detection from the CLI is as simple as running `pulumi refresh --preview-only` or even just `pulumi refresh`, which will first run a preview (thereby creating the drift run), before running the actual refresh.
+Any preview of a refresh is considered a drift detection run in Pulumi Cloud. Running Drift Detection from the CLI is as simple as running `pulumi refresh --preview-only` or even just `pulumi refresh`, which will first run a preview (thereby creating the drift run), before running the actual refresh.
```

Comment:
Is drift detection a proper noun?

Reply:
Products are proper nouns (Pulumi Deployments), features/capabilities should not be.